completions/puppet

```diff
@@ -101,16 +101,16 @@ _puppet()
             ;;
         puppet)
             case ${words[1]} in
-                agent|apply|cert|describe|doc|filebucket|kick|lookup|master|parser|queue|resource)
+                agent|apply|catalog|cert|describe|doc|epp|filebucket|help|kick|lookup|man|master|module|node|parser|report|resource)
                     subcommand=${words[1]}
```

Comment:
This seems to add more than just epp, please adjust the commit message accordingly.

Also, I suggest splitting the commit to two, one for adding commands and another removing them.